Providers/nxOMSContainers/MSFT_nxOMSContainersResource.schema.mof

```diff
@@ -0,0 +1,6 @@
+[ClassVersion("1.0.0")]
+class MSFT_nxOMSContainersResource : OMI_BaseResource
+{
+    [key] string ResourceSettings;
```

Comment:
I would much rather you have separate fields, one a WorkspaceId. But this approach is up to you.